docs/proposals/20220215-obsctl.md

```diff
@@ -0,0 +1,77 @@
+# obsctl: A CLI to interact with Observatorium
+
+* **Owners:**
+  * `@onprem` `@saswatamcode`
+
+* **Related Tickets:**
+  * https://issues.redhat.com/browse/MON-2199
+
+* **Other docs:**
+  * None
+
+## TL;DR
+
+We can interact with [Observatorium API](https://github.com/observatorium/api) with a convenient and dedicated CLI, which provides capabilities to persist auth data, and allows us to easily & quickly get up and running with our operations.
+
+## Pitfalls of Current Solution
+
+Currently, we do not have a simple and convenient way to interact with the Observatorium API from the command line. Manually crafting cURL commands while authenticating with OIDC or mTLS is not an optimal experience and generally confusing/trickier for users, both old and new.
+
+## Goals
+
+obsctl will,
+* Manage authentication configuration for multiple tenants by saving them locally
+* Allow users to switch between tenants conveniently
+* Configure and view rules for a tenant
+* View series, rules, and labels for a tenant
+* Later on, support more such one-off operations for other domains as well (logs, traces, alert-routing configs)
+
+## Non-Goals
+
+* Enabling Observatorium administration via obsctl
```

Comment:
what does it mean 'Observatorium administration'? Perhaps we could clarify this better?

Reply:
does it refer to https://github.com/observatorium/observatorium/pull/456/files#r807905979 for example?

Reply:
Sure, let me clarify! 🙂

Reply:
Clarified this here: https://github.com/observatorium/observatorium/pull/456/commits/16ccd1d1c2fd096def38d20b2322960042c08b02.